src/components/progress.scss

```diff
@@ -0,0 +1,133 @@
+// * Progress Bar component
+// *
+// *
+
+.ProgressBar-range {
+  height: $su-small;
+  flex: 1; // for inline style
+  background: $cu-middleground--dark;
+  box-shadow: inset $box-shadow-small;
+  border-radius: $border-radius-large;
+  margin-bottom: $su-small;
+  .ProgressBar--large & {
+    height: $su-default;
+  }
+  .ProgressBar--small & {
+    height: $su-xsmall;
+  }
+  .ProgressBar--extra-small & {
```

Comment:
These modifiers should be camel case or perhaps reduced to one word. https://teamsnap-ui-patterns.netlify.com/docs/naming-convention.html

ie `ProgressBar--extraSmall` or if it's treated as just one word it could be `ProgressBar--xsmall`

Reply:
Oh gotcha! I think I copied over the naming style from an old component. I will keep this in mind going forward. I will do xsmall and xlarge!